spec/components/show/controls_component_spec.rb

```diff
@@ -90,6 +116,7 @@
   describe '#registered_only?' do
     subject { component.send(:registered_only?) }
 
+    let(:allows_modification) { true }
     let(:item_id) { 'druid:kv840xx0000' }
 
     context 'when registered' do
```

Comment:
note: this spec looks like is has more changes that it actually does ... basically I created new context blocks for the DRO so that we can verify the embargo buttons are enabled or disabled as needed based on the locked/unlocked status ... the indenting caused a big diff